tests/parser/TableManagerTests.cpp

```diff
@@ -651,8 +657,635 @@ BOOST_AUTO_TEST_CASE(FoammobTable_Tests) {
     }
 }
 
+BOOST_AUTO_TEST_CASE(PvdoTable_Tests) {
+    // PVDO tables from opm-tests/model6/0_BASE_MODEL6.DATA .
+    const auto deck = Opm::Parser{}.parseString(R"(RUNSPEC
+OIL
+WATER
+TABDIMS
+1 2 /
+PROPS
+DENSITY
+   924.1      1026.0      1.03446 /
+   924.1      1026.0      1.03446 /
+PVDO
+ 23.0  1.10770  52.630
+ 27.5  1.08610  53.660
+ 32.1  1.06460  54.730
+ 50.0  1.06350  58.940
+/
+/ -- Copied from table 1
+END
+)");
+
+    const auto tmgr = Opm::TableManager { deck };
+    const auto& pvdo = tmgr.getPvdoTables();
+    BOOST_REQUIRE_EQUAL(pvdo.size(), std::size_t{2});
+
+    {
```

Comment:
> Seems like we expect the same values for both tables.

Yes.  We're testing that the table copying works.

> Any particular reason not to factor out the check into a function and call this twice?

Putting `BOOST_CHECK` assertions in a function means that the reporting system's line numbers will come from that function instead of the call site which makes it (marginally) harder to track down the failure if something goes wrong.